command/connect/envoy/envoy.go

```diff
@@ -117,6 +118,10 @@ func (c *cmd) init() {
 	c.flags.StringVar(&c.wanAddress, "wan-address", "",
 		"WAN address to advertise in the Mesh Gateway service registration")
 
+	c.flags.Var((*flags.FlagMapValue)(&c.bindAddresses), "bind-address", "Bind "+
+		"addresses to use instead of the default binding rules given as `<name>=<ip>:<port>` "+
```

Comment:
nit: slightly confusing that the doc for `-bind-address` described is as the "bind adresses" (plural).
```suggestion
		"address to use instead of the default binding rules given as `<name>=<ip>:<port>` "+
```